lib/logflare/backends.ex

```diff
@@ -428,15 +428,28 @@ defmodule Logflare.Backends do
     with %Backend{} = backend <- get_backend(backend_id) do
       sources = Sources.list_sources(backend_id: backend_id)
 
-      for source <- sources do
-        Cluster.Utils.rpc_multicall(SourceSup, :start_backend_child, [source, backend])
-        Cluster.Utils.rpc_multicall(__MODULE__, :clear_list_backends_cache, [source.id])
+      if length(sources) > 0 do
```

Comment:
doesn't matter for small lists but instead of calculating the length of the list just to compare with zero, it's faster to compare the list with empty one:

```suggestion
      if sources != [] do
```